From 0062d3b699cd061b4aa50a36261aafd75d10b7f5 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Fri, 6 Jan 2017 13:53:44 +0800
Subject: [PATCH 1463/5242] MLK-13729-1 ARM: imx: add the reg into the rpmsg
 node

commit  5ef6392dc4b14a74fcd885a570c520ed7bb85e20 from
https://source.codeaurora.org/external/imx/linux-imx.git

In order to remove the hard-coded vring buffer in
the driver, input the vring buffer by device tree
node.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb-m4.dts |    2 ++
 arch/arm/boot/dts/imx7d-sdb-m4.dtsi |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-m4.dts b/arch/arm/boot/dts/imx6sx-sdb-m4.dts
index 7b67b8e..61a1db3 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-m4.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-m4.dts
@@ -57,6 +57,8 @@
 };
 
 &rpmsg{
+	vdev-nums = <1>;
+	reg = <0xbfff0000 0x10000>;
 	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/imx7d-sdb-m4.dtsi b/arch/arm/boot/dts/imx7d-sdb-m4.dtsi
index 1b29fc3..deba9ef 100644
--- a/arch/arm/boot/dts/imx7d-sdb-m4.dtsi
+++ b/arch/arm/boot/dts/imx7d-sdb-m4.dtsi
@@ -54,6 +54,8 @@
 };
 
 &rpmsg{
+	vdev-nums = <1>;
+	reg = <0xbfff0000 0x10000>;
 	status = "okay";
 };
 
-- 
1.7.9.5

